src/Symfony/Component/Routing/Matcher/RedirectableUrlMatcher.php

```diff
@@ -32,9 +32,9 @@ public function match($pathinfo)
             }
 
             try {
-                parent::match($pathinfo.'/');
+                $parameters = parent::match($pathinfo.'/');
 
-                return $this->redirect($pathinfo.'/', null);
+                return array_replace($parameters, $this->redirect($pathinfo.'/', isset($parameters['_route']) ? $parameters['_route'] : null));
```

Comment:
right, @nicolas-grekas can you help @tobion with the best composer changes to do here?